heroku-20/rds-global-bundle.pem

```diff
@@ -0,0 +1,2528 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This looks like the same file copied 3 times? Is there any way to put it into the repo only once? Maybe a `certs` or `share` directory at the root?

Reply:
I believe the Dockerfile context root is the stack subdirectory, so using a shared asset outside of that wouldn't be possible without some restructuring (which I'm not against) - which is why I added the Postgresql APT key assets multiple times in #256. Git does de-dupe identical files within its store, so it won't bloat the repo at least.